change/@fluentui-react-c8535f3e-253b-40b8-92e7-bb6c8e552afe.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29766&commitId=d83e1a0a53d2d24590abd6b4b0f89bf5ec363760&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Rendering callout attached to a rectangle.default.chromium.png|1604386| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Multiple - Dark Mode.hover.chromium.png|4360| Changed|
|react-charting-AreaChart.Multiple.hover.chromium.png|4169| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>